### DISCUSSION TITLE
More Future Mobs

### DISCUSSION THREAD

**Discussion by SuperMariobrine** [Roadhog360/Et-Futurum-Requiem#151] [Ideas]
As of writing this, there are very few mods that attempt to backport mobs from the later versions, spanning from 1.8 to 1.14. Ignoring 1.16 since Netherlicious has that covered, and it's too soon to backport 1.19 things, I don't even think it's possible at the moment. I really like the illagers and the Killer Bunny and I'm surprised that nobody has tried to backport them to 1.7.10 yet, seeing that you backported the Shulkers gave me hope. I've seen a bunch of mods that backport features from later versions, but they do blocks and items for the most part which we have seen several times. I want to see more mods that backport later content to 1.7.10 that other mods haven't done yet.

Here are a list of mobs I request that you implement:
Add the illagers, (Vindicator, Evoker, Vex, Illusioner), idk about the mansion that might be hard.
Add the raiders, (Pillager and Ravager, and maybe the raid function if that's possible), the outpost should be easier at least.
Add the Killer Bunny (make it another struck by lightning event)
Add the Skeleton Horseman (whether it'll be one or three of them at once is up to you)
Add the Llama and Turtle (They'll spawn in their respective biomes from the higher versions)
Add the Drowned (Maybe you could add the Ocean Ruins, to spawn in the deep ocean)
Add the Wandering Trader (Keep them as reclusive as they are in 1.14, and don't change their trades too much)

I know there are other mobs and animals but some of them I would take more work into giving them a suitable environment, some other mods already add the mobs I didn't list, and some of them I'm just not that interested in.

**Comment by Roadhog360** [comment]
There are few mobs because they are probably one of the most difficult things to backport alongside structures. I must be very selective with which ones get ported.

**Reply by SuperMariobrine** [reply]
Yeah I understand that, that's why I didn't list some animals like the Axolotl, Polar Bear, Parrot, and Dolphin. I'm not going to be one of those people who immediately ask for the most recent addition added to Minecraft to be backported like the Warden and Allay. But I have always wanted the illagers in 1.7.10 so I want those guys over anything else. Doesn't matter to me if it takes forever because I know it'll be worth waiting for, as long as its implemented wholeheartedly.

**Comment by syndicate25** [comment]
I would second Illagers as a single choice from this list, if you had to/were going to pick one.  That's not exactly one mob, since I know there's quite a few different ones, but if it can be done... it would be awesome!

**Comment by Roadhog360** [comment]
Closing due to the broad-ness of the request.

**Reply by SuperMariobrine** [reply]
I agree, I realized upon re-reading it that I listed too much. I did point out that the animals shouldn't be the biggest priority, since there are several mods that already add those animals. Of all the mobs, I would focus the Illagers the most since there are no mods that backported them, also the Village Names mod has an item that correlates to the Woodland Mansion should a mod backport the Illagers with that. Also the Killer Bunny could work if the Illager part is too taxing, I actually think bringing a rabbit to the nether should change it to Killer Bunny instead of lightning, makes it sound more demonic.

**Reply by Roadhog360** [reply]
Adding the killer bunny to the nether is out of scope for the mod.

**Reply by GregoriusT** [reply]
yeah not only would it be out of scope it would also ruin anyones hope of transferring Bunnies through the Nether back to their base, or have a Base in the Nether in the first place if they like to have Buns as Pets.

**Reply by SuperMariobrine** [reply]
Then perhaps it could be created by throwing a night vision potion on a rabbit and feeding it a beetroot, then after 1-2 minutes it would turn into a Killer Bunny. Is this pitch better?

**Reply by Roadhog360** [reply]
It's not vanilla so no. This suggestion would be better for a future mod I'm working on yet to be announced. Only very specific and small tweaks get into tweaks.cfg
